go/vt/vttablet/tabletserver/incomingquerythrottler/incoming_query_throttler.go

```diff
@@ -0,0 +1,131 @@
+package incomingquerythrottler
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"vitess.io/vitess/go/vt/log"
+	querypb "vitess.io/vitess/go/vt/proto/query"
+	topodatapb "vitess.io/vitess/go/vt/proto/topodata"
+	"vitess.io/vitess/go/vt/vttablet/tabletserver/throttle"
+	"vitess.io/vitess/go/vt/vttablet/tabletserver/throttle/base"
+	"vitess.io/vitess/go/vt/vttablet/tabletserver/throttle/throttlerapp"
+)
+
+var (
+	_configRefreshTicket = time.NewTicker(1 * time.Minute)
+)
+
+type IncomingQueryThrottler struct {
+	ctx            context.Context
+	throttleClient *throttle.Client
+	mu             sync.RWMutex
+	// cfg holds the current configuration for the throttler.
```

Comment:
We should not use the global ticker here and pass the timer in constructor. 
This way we will also remove the usage of gostub in the test.